hangzhounet/values.yaml

```diff
@@ -0,0 +1,149 @@
+    key: edpkuu3jmS3kbaLYTmyVaPHaxowC1KP8fDSrLoJhcvip1Cm2tK5QCd
+    type: public
+    is_bootstrap_baker_account: true
+    bootstrap_balance: "5000000000000"
+  tezosil:
+    # unconfirmed
+    # tz1VWasoyFGAWZt5K2qZRzP3cWzv3z7MMhP8
+    key: edpkvQj8r3YigN6QFsDRaKQCoxDNhKK4whDUkFkw6Gom36d86uhEbv
+    type: public
+    is_bootstrap_baker_account: true
+    bootstrap_balance: "5000000000000"
+  pirbo:
+    # unconfirmed
+    # tz1NFs6yP2sXd5vAAbR43bbDRpV2nahDZope
+    key: edpkuGH2TWTQ5JrwzBLGRu5is1h7vcv5e5LN45VfqmXS5zaTUMpopH
+    type: public
+    is_bootstrap_baker_account: true
+    bootstrap_balance: "5000000000000"
+  marco:
+    # unconfirmed
+    # tz1T8UYSbVuRm6CdhjvwCfXsKXb4yL9ai9Q3
+    key: edpkvMWHXeETjH4i2h2ngkG2y6aVdgtW99pQ3SuHJoWNEPKANzVRLT
+    type: public
+    is_bootstrap_baker_account: false
+    bootstrap_balance: "5000000000000"
+  tz1THUNA:
+    # tz1THUNARo58aD5YdNGYPnWNnvd8yHPrMdsF 
+    # Romain's faucet
+    key: edpkuh3WSwziyd6nXqDXeh48vZHp5ETxFd43EdkbzhmW98BvuLXK81
+    type: public
+    is_bootstrap_baker_account: false
+    bootstrap_balance: "500000000000000"
+  oxheadbaker:
+    # Oxhead Alpha's baker hosted in teztnets.xyz infra
+    # tz1foXHgRzdYdaLgX6XhpZGxbBv42LZ6ubvE
+    bootstrap_balance: "5000000000000"
+    is_bootstrap_baker_account: true
+    type: secret
+    # key injected by pulumi
+  oxheadfree:
+    # an account with a lot of money in it
+    # tz1foxFdz2ieSj8P9YxKYBTEqYbgFXXEeiQY
+    key: edpkubSgCoMjB2a6vPWRNRHfGa6HUFfFyEUKoEQhWC8osnav8QzBdJ
+    bootstrap_balance: "5000000000000000"
+    is_bootstrap_baker_account: false
+    type: public
+
+node_config_network:
+  activation_account_name: oxheadbaker
+  chain_name: TEZOS_HANGZHOUNET_2021-09-22T15:00:00Z
+  genesis:
+    block: BLockGenesisGenesisGenesisGenesisGenesisd4299hBGVoU
+    protocol: PtYuensgYBb3G3x1hLLbCmcav8ue8Kyd2khADcL5LsT5R1hcXex
+    timestamp: "2021-09-22T15:00:00Z"
+  user_activated_upgrades:
+    - level: 4095
+      replacement_protocol: PtHangzHogokSuiMHemCuowEavgYTP8J5qQ9fQS793MHYFpCY3r
+
+bootstrap_peers:
+  - hangzhounet.smartpy.io
+  - hangzhounet.tezos.co.il
+  - hangzhounet.kaml.fr
+
+images:
+  octez: tezos/tezos:v11.0-rc1
+
+nodes:
+  tezos-baking-node:
+    instances:
+      - bake_using_accounts:
+          - oxheadbaker
+        config:
+          shell:
+            history_mode: archive
+          rpc:
+            cors-headers: [ "content-type" ]
+            cors-origin: [ "*" ]
+        is_bootstrap_node: true
+    runs:
+      - baker
+      - endorser
+      - octez_node
+    storage_size: 30Gi
+  # Overwrite default values.yaml tezos-node
+  tezos-node: null
+
+activation:
+  protocol_hash: PtGRANADsDU8R9daYKAgWnQYAJ64omN1o3KMGVCykShA97vQbvV
+  protocol_parameters:
+     preserved_cycles: 3
+     blocks_per_cycle: 4096
+     blocks_per_commitment: 32
+     blocks_per_roll_snapshot: 256
+     blocks_per_voting_period: 20480
+     time_between_blocks:
+       - "10"
+       - "20"
```

Comment:
Shouldn't it be the other way around?

Reply:
yes, thanks. although it's not clear what this does since "minimal_block_delay" seems to be deciding how much time between blocks now.